packages/jest-config/src/defaults.js

```diff
@@ -63,6 +63,7 @@ module.exports = ({
   testURL: 'about:blank',
   timers: 'real',
   transformIgnorePatterns: [NODE_MODULES_REGEXP],
+  transformPlugins: [],
```

Comment:
I added `transformPlugins: []` to wherever I could. — If I need to remove some of them, or add it to more places please let me know.